app/assets/javascripts/controllers/ems_infra_dashboard/aggregate_status_card_controller.js

```diff
@@ -0,0 +1,72 @@
+/* global miqHttpInject */
+
+angular.module( 'patternfly.card' ).controller('aggregateStatusCardController', ['$q', 'providerId', 'API', 'miqService', function($q, providerId, API, miqService) {
+  var vm = this;
+  var attributes = ["ems_clusters", "hosts", "storages", "vms", "miq_templates"];
+  var attrHsh = {
+    "ems_clusters": "Clusters",
+    "hosts": "Hosts",
+    "storages": "Datastores",
+    "vms": "VMs",
+    "miq_templates": "Templates",
+  };
+
+  var attrIconHsh = {
+    "ems_clusters": "pficon pficon-cluster",
+    "hosts": "pficon pficon-screen",
+    "storages": "fa fa-database",
+    "vms": "pficon pficon-virtual-machine",
+    "miq_templates": "pficon pficon-virtual-machine",
+  };
+
+  var attrUrl = {
+    "ems_clusters": "ems_clusters",
+    "hosts": "hosts",
+    "storages": "storages",
+    "vms": "vms",
+    "miq_templates": "miq_templates",
+  };
+
+  var init = function() {
+    ManageIQ.angular.scope = vm;
+    var promiseProviderData = API.get("/api/providers/" + providerId + "?attributes=" + attributes)
+      .then(function(data) {
+        vm.provider = data;
+      })
+      .catch(miqService.handleFailure);
+
+    $q.all([promiseProviderData]).then(function() {
+      vm.status = {
+//         "title": vm.provider.name,
+        "iconImage": "/assets/svg/vendor-" + getIcon(vm.provider.type) + ".svg",
```

Comment:
This does not work outside the development enrionment.

Image paths must be created using the `image_path` helper, ruby-side